### PR TITLE
[WIP] Tensorflow frontend random uniform

### LINKED ISSUE #6183: uniform

### PR DESCRIPTION
**Pull request by ItIsUday**
Close #6183 
I'm unsure of how to proceed with writing tests for minval and maxval with tensors as input. Any help is much appreciated. Thanks

### CODE REVIEW

**Review by simonetgordon — CHANGES_REQUESTED** [review]
Hi there,

I think you just need to have the argument 'dtype' as input to the test helper function, not all the individual input dtypes.

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_random.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        with_out=with_out,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        frontend="tensorflow",
+        fn_tree="random.uniform",
+        shape=shape,
+        minval=minval[0],
+        maxval=maxval[0],
+        seed=seed[0],
+        dtype="numeric",
```
Comment (ItIsUday) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_random.py:57]: I'm not sure why the tests fail when dtype receives bounded numeric values from get_dtypes.

Comment (simonetgordon) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_random.py:57]: Hi there, the `dtype` input to the helper function should be a data type and not a string dype. You may want to think about how the output `dtype` (which is what the `dtype` inputted to the function will be) relates to the dtypes of the inputs. You don't want any type promotion errors occuring.

**Review by simonetgordon — CHANGES_REQUESTED** [review]

**Comment by simonetgordon** [comment]
@ItIsUday Hi there, are you still working on this PR?

**Comment by ItIsUday** [comment]
Hey, I'm occupied elsewhere and I don't have the bandwidth to complete it anytime soon. Feel free to close the PR.

**Pull request closed without merging**